editors/vscode/pnpm-lock.yaml

```diff
@@ -1,7 +1,7 @@
 lockfileVersion: '6.0'
 
 settings:
-  autoInstallPeers: true
+  autoInstallPeers: false
```

Comment:
How is this being changed? I'm not seeing any other config changes related to this.